src/components/ComputeModal.js

```diff
@@ -624,6 +634,10 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
     }
   }
 
+  const copyImageLinkTooltipText = 'Copy the image version'
+  const isTerraSupported = runtime => (!runtime.isCommunity && !(getToolForImage(runtime.id) === tools.RStudio.label))
+  const getChangelogUrl = runtime => `https://github.com/DataBiosphere/terra-docker/blob/master/${runtime.id.replace('_legacy', '')}/CHANGELOG.md`
```

Comment:
```suggestion
  const getChangelogUrl = ({ id }) => `https://github.com/DataBiosphere/terra-docker/blob/master/${_.replace('_legacy', '', id)}/CHANGELOG.md`
```

We usually prefer Lodash FP functions over native functions for several reasons:

- Lodash will handle the case where you try to run the function on an incorrect type. e.g.: null, undefined, Object  (in the case of slice)
{}.slice(2,4) // throws an error
_.slice(2, 4, {}) // returns an empty array
- When running through your code, if you pulled data in from an outside source and got unexpected garbage, the application will not have an error thrown, and can continue along with the value / type it was expecting when using Lodash.
- Lodash/fp functions support currying. This is especially useful for code changes later. If someone wants to add functionality, it makes it easy to do something like add the function to a flow.
- Lodash/fp will not mutate an array. In the case of native slice, it also returns a new array. But there are other functions that can mutate (e.g. push/pop)
- For consistency within the codebase, consistency with how the code operates, and to prevent mutation we tend to prefer using lodash functions.
- Mixing native and lodash functions can lead to confusion about behavior.